### PR TITLE
update and clean code per new repo features (thanks @martyn :))

### PR DESCRIPTION
**Pull request by adam-saland**
TODO:

- [x] Default Styles

### CODE REVIEW

File: how-to/use-channel-api/issue-commands-to-a-platform/public/html/app.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Launched View</title>
-		<link rel="stylesheet" href="../../common/style/app.css" />
+		<link rel="stylesheet" href="http://localhost:5050/common/style/app.css" />
```
Comment (obany) [inline how-to/use-channel-api/issue-commands-to-a-platform/public/html/app.html:8]: You should be able to set this to `href="/common/style/app.css` the deployment script will update this accordingly

File: how-to/use-interop/setup-multi-platform-interop/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Use Cross Platform Interop
+
+This repository demonstrates how to use OpenFin's interop overrides to enable context sharing from two different Platform Applications
+
+Before diving in we recommend taking some time familiarize yourself with the concepts and terms found in the [interoperability overview](https://developers.openfin.co/of-docs/docs/interoperability-overview) section of our documentation.
+
+---
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/README.md:7]: Remove `---` separator, not necessary before heading

File: how-to/use-interop/setup-multi-platform-interop/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Before diving in we recommend taking some time familiarize yourself with the concepts and terms found in the [interoperability overview](https://developers.openfin.co/of-docs/docs/interoperability-overview) section of our documentation.
+
+---
+
+## How it Works
+
+The interop object, made available through OpenFin views and windows, is programmatically accessible through the concept of platform provider overrides. A platform provider override enables a developer to implement additional logic from objects defined on an instance of an OpenFin window or view entity. In this case specifically, we will be focusing on the `interopOverride` implementing additional logic related to the `interop` object on a Platform's window and view entities.
+
+---
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/README.md:13]: Remove `---` separator, not necessary before heading

File: how-to/use-interop/setup-multi-platform-interop/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+## Get Started
+
+Follow the instructions below to get up and running.
+
+### Set up the project
+
+1. Install dependencies. Note that these examples assume you are in the sub-directory for the example.
+
+```bash
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/README.md:23]: Markdown shell should be
<pre>
```shell
npm install
```
</pre>
i.e. change bash to shell and remove `$` on command

File: how-to/use-interop/setup-multi-platform-interop/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+		"start": "npm run server",
+		"platform1": "start fin://localhost:5050/config/platform1.json",
+		"platform2": "start fin://localhost:5050/config/platform2.json",
+		"server": "node ./server/build/index.js",
+		"client": "npm --prefix ../../common/ run client",
+		"kill": "npm --prefix ../../common/ run kill",
+		"setup": "cd ../../../ && npm install && cd how-to/use-interop/setup-multi-platform-interop && npm run build",
+		"validate": "npm --prefix ../../../ run validate"
+	},
+	"license": "ISC",
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/package.json:19]: `license` should be `"license": "SEE LICENSE IN LICENSE.MD"`

Looks like a lot of the other packages need updating too!

File: how-to/use-interop/setup-multi-platform-interop/package.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+	"dependencies": {
+		"express": "^4.18.1",
+		"@openfin/core": "latest",
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/package.json:22]: Fix this to specific version

File: how-to/use-interop/setup-multi-platform-interop/package.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+		"openfin-adapter": "^25.69.8",
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/package.json:23]: Remove `openfin-adapter`

File: how-to/use-interop/setup-multi-platform-interop/public/html/provider.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/public/html/provider.html:1]: Does this need styling ?

**Review by obany — CHANGES_REQUESTED** [review]
Few minor changes

**Review by obany — APPROVED** [review]

**Review by johnman — CHANGES_REQUESTED** [review]

File: how-to/use-interop/setup-multi-platform-interop/README.md (hunk 4 above)
Comment (johnman) [reply how-to/use-interop/setup-multi-platform-interop/README.md]: Should this be the new npm run setup command?

File: how-to/use-interop/setup-multi-platform-interop/client/src/window.ts [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+	})
+	.catch((error) => console.error(error));
+
+const changeContextGroup = async (event: Event): Promise<void> => {
+	const selectedColorElement: HTMLElement = event.target as HTMLElement;
+	const color: string = selectedColorElement.title;
+	await fin.me.interop.joinContextGroup(color, lastFocusedView);
+	document
+		.querySelector(`#tab-${lastFocusedView.name}`)
+		.classList.remove(
```
Comment (obany) [inline how-to/use-interop/setup-multi-platform-interop/client/src/window.ts:22]: Can we use getContextGroups and loop through them to remove the classes, something like

```
const contextGroups: PlatformContextGroups = await fin.me.interop.getContextGroups();
document
		.querySelector(`#tab-${lastFocusedView.name}`)
		.classList.remove(...contextGroups.map(channel => `${channel.displayMetadata.name}-channel`))
```

File: .eslintrc.js [hunk 10]
```diff
@@ -405,7 +405,7 @@ module.exports = {
 				'@typescript-eslint/consistent-type-assertions': ['error'],
 				'@typescript-eslint/consistent-type-definitions': ['error'],
 				'@typescript-eslint/default-param-last': ['error'],
-				'@typescript-eslint/dot-notation': ['error'],
+				'@typescript-eslint/dot-notation': ['off'],
```
Comment (obany) [inline .eslintrc.js:408]: This would be better disabled where the fix is required, with a comment as to why if its not self explanatory

**Review by obany — CHANGES_REQUESTED** [review]
Couple of tweaks, and remove the package-lock.json

**Review by obany — APPROVED** [review]

**Pull request merged**